CONTRIBUTING.md

```diff
@@ -131,13 +132,33 @@ then run the server:
 make start
 ```
 
-This will start the server using SQLite as database. If you want to run with Cassandra and Elasticsearch, then run these commands:
+This will start the server using SQLite as an in-memory database. You can choose other databases as well.
+
+If you want to run with Cassandra and Elasticsearch, then run these commands:
```

Comment:
Probably just move `make start-dependencies` here and restructure the doc a bit.